code/modules/antagonists/ert/ert.dm

```diff
@@ -12,6 +12,8 @@
 	antagpanel_category = ANTAG_GROUP_ERT
 	suicide_cry = "FOR NANOTRASEN!!"
 	count_against_dynamic_roll_chance = FALSE
+	// Not 'true' antags, cannot induct to prevent issues
+	antag_flags = null
```

Comment:
```suggestion
	antag_flags = NONE
```

don't set bitflag vars to `null`, that might cause yonkiness. `NONE` is just `0`.

Reply:
yeah i wanted to use NO_FLAGS but it wasnt named that on this codebase